### ISSUE TITLE
No module named 'spacy'' error

### ISSUE THREAD

**Issue by acrigney** [henon/Python.Included#60]
Sorry guys I am getting this error I must not have my Python environment setup properly?
I am new to this I just tried to run the .net core example, and I get the same error upgrading it to .net 7
```csharp
        static async Task Main(string[] args)
        {
            // This example demonstrates how Python.Included is able to automatically install a minimal Python
            // environment which it includes as an embedded resource in its .NET assembly file
            // Python.Included is currently fixed to Python 3.7.3 amd64 for windows
            // If you need a different Python version or platform check out the Python.Installer examples!

            // install in local directory
            Installer.InstallPath = Path.GetFullPath(".");

            // see what the installer is doing
            Installer.LogMessage += Console.WriteLine;

            // install the embedded python distribution
            await Installer.SetupPython();

            // install pip3 for package installation
            Installer.TryInstallPip(true);

            // download and install Spacy from the internet
            Installer.PipInstallModule("spacy");

            // ok, now use pythonnet from that installation
            PythonEngine.Initialize();

            // call Python's sys.version to prove we are executing the right version
            dynamic sys=Py.Import("sys");
            Console.WriteLine("### Python version:\n\t" + sys.version);

            // call os.getcwd() to prove we are executing the locally installed embedded python distribution
            dynamic os = Py.Import("os");
            Console.WriteLine("### Current working directory:\n\t" + os.getcwd());
            Console.WriteLine("### PythonPath:\n\t" + PythonEngine.PythonPath);

            // call spacy
            dynamic spacy = Py.Import("spacy"); // throws No module named 'spacy''
            Console.WriteLine("### Spacy version:\n\t" + spacy.__version__);

            Console.WriteLine("\nDone. Press any key to exit.");
#if NETFRAMEWORK
            Console.ReadKey();
#endif
        }
```

**Comment by henon** [comment]
Try installing spacy from the command line in the directory where Python.Included installed the embedded python distribution and see if it works. If not you know more. 

If it works manually then you'd have to debug with the Python.Included source code to find out what's going on.

**Comment by acrigney** [comment]
Thanks mate but isn't the framework supposed to do the install when you
call the install function?

**Comment by henon** [comment]
Yes, I found the error. It was a bug in `RunCommand` which is called by `PipInstallModule` and caused pip not to unstall any libraries. It is fixed now.

**Comment by henon** [comment]
https://www.nuget.org/packages/Python.Deployment/2.0.5
https://www.nuget.org/packages/Python.Included/3.11.6
